src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -696,6 +702,8 @@ private void UpdateRenderTreeToMatchClientState(ulong eventHandlerId, EventField
         /// <param name="disposing"><see langword="true"/> if this method is being invoked by <see cref="IDisposable.Dispose"/>, otherwise <see langword="false"/>.</param>
         protected virtual void Dispose(bool disposing)
         {
+            _disposed = true;
```

Comment:
Should we no-op everything after this if this has already been disposed? 

```C#
if (_disposed)
{
 return;
}

_disposed = true;
```

I realize everything after is safe from double-disposal (we clear the dictionary and the builder allows double-Dispose), but this makes it a lot clearer